synth.py

```diff
@@ -5,46 +5,53 @@
 import os
 
 logging.basicConfig(level=logging.DEBUG)
-
-gapic = gcp.GAPICGenerator()
-v2_library = gapic.node_library(
+version='v2'
+gapic = gcp.GAPICMicrogenerator()
+v2_library = gapic.typescript_library(
   'bigtable',
   'v2',
```

Comment:
if you have `version='v2'`, let's use it here :)

Reply:
sure!